### PR TITLE
fix: Fix package.py commands after :zip not being executed

### PR DESCRIPTION
**Pull request by mleziva**
## Description
When using the commands option in source_path, commands after ":zip" were not being executed. This PR resolves that unexpected behavior

## Motivation and Context
The root cause was that the commands loop read all provided commands, but it didn't add the commands after a ":zip" command to the plan file. This was remediated by adding a check after the commands loop exited to add the remaining commands to the steps. 

## Breaking Changes
None

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Comment by mleziva** [comment]
@antonbabenko , can you please review when you get a chance. Thanks!

File: tests/fixtures/node-app/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (antonbabenko) [inline tests/fixtures/node-app/package-lock.json:1]: Do we need to have this file in the examples? If not, let's simplify it and drop it.

**Review by antonbabenko — CHANGES_REQUESTED** [review]
LGTM.

@pdecat what do you think?

File: tests/fixtures/node-app/package-lock.json (hunk 1 above)
Comment (mleziva) [reply tests/fixtures/node-app/package-lock.json]: It is required for the `npm ci` command, but I just updated the command to be `npm install` and removed this file to simplify it

**Review by antonbabenko — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 7.8.1 :tada:

**Pull request merged**